packages/dev-server/README.md

```diff
@@ -28,6 +28,13 @@ module.exports = createConfig([
 ```
 
 
+## Pitfalls
+
+Make sure you use this `dev-server` block after your `entryPoint()` blocks. The dev server will extend all of them with the `'webpack/hot/only-dev-server'` entry point by default.
```

Comment:
Now that it is normalized webpack-merge is taking care of merging everything correctly. It shouldn't be a problem if the dev-server is before or after the entry point.